cmd/agent/Dockerfile.buildx

```diff
@@ -3,9 +3,9 @@ ARG TARGETPLATFORM
 ARG BUILDPLATFORM
 
 # Use custom Go version instead of one prepacked in seego
-ENV GOLANG_VERSION 1.17.6
+ENV GOLANG_VERSION 1.18
```

Comment:
Annoyingly the Docker container is 1.18.0 but the version on their downloads page is just 1.18